Remotion/Data/DomainObjects/Mapping/Validation/Logical/SortExpressionIsSupportedForCardinalityOfRelationPropertyValidationRule.cs

```diff
@@ -54,7 +54,7 @@ private MappingValidationResult Validate (IRelationEndPointDefinition relationEn
             virtualObjectRelationEndPointDefinition.PropertyInfo,
             "Property '{0}' of class '{1}' must not specify a SortExpression, because cardinality is equal to 'one'.",
```

Comment:
```suggestion
            "Property '{0}' of type '{1}' must not specify a SortExpression, because cardinality is equal to 'one'.",
```

Reply:
actually, please check all validators, there's other "class" references in the messages, too, even though the validation rule isn't specfic to classdefinition